### PR TITLE
Add gitpod config

### PR DESCRIPTION
**Pull request by BacLuc**
Try it out yourself at: https://www.gitpod.io/#https://github.com/BacLuc/jumpstart-docs/tree/add-gitpod-config

The first start takes a while, when you reenter a gitpod it's fast. (https://gitpod.io/workspaces)
After 30 min idle time, the gitpod is automatically stopped.

You have 50h per month for free at gitpod, which is enough for the 7 x 4h course.

### CODE REVIEW

**Comment by BacLuc** [comment]
Should i merge and use it for the testing part?

File: gitpod/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        wget \
+        # needed for ansible
+        python3-pip \
+        # needed for drawio-desktop
+        desktop-file-utils \
+        xvfb \
+    && sudo rm -rf /var/lib/apt/lists/* \
+    && sudo pip --no-cache-dir install ansible
+
+COPY --from=ghcr.io/langchr86/docmake:latest /tmp/ansible-role-docmake /tmp/ansible-role-docmake
```
Comment (langchr86) [inline gitpod/Dockerfile:23]: Would be more flexible to clone the origin repo (https://github.com/langchr86/ansible-role-docmake) and execute the script from there.

**Comment by langchr86** [comment]
Looks good.

File: gitpod/Dockerfile (hunk 1 above)
Comment (BacLuc) [reply gitpod/Dockerfile:23]: I prefer to copy things from docker container, because they can be cached.
(and have defined version).

**Pull request merged**